src/components/Form/Form.js

```diff
@@ -0,0 +1,42 @@
+import dataParser from 'src/model/dataParser'
+import rowTemplate from './row.tpl.html'
+import { newIndexedArray } from 'src/utils/array'
+import { htmlToElement } from 'src/utils/dom'
+
+const TEMPLATE = `<div class="vertical-container"></div>`
+
+const render = element => {
+  element.innerHTML = ''
+  const main = htmlToElement(TEMPLATE)
+  newIndexedArray(5).forEach(() => {
```

Comment:
@francesco-strazzullo I don't understand why you use this function (`newIndexedArray`) instead of a simple `for` cycle

Reply:
It's to keep as much as possible a function approach. It really useful in the math part of the code where I preferred this way to keep composing at an high level. At that point I decided to use always the same pattern.

Reply:
ok